docker-compose.yml

```diff
@@ -0,0 +1,5 @@
+version: "3.9"
```

Comment:
I would leave a comment and explain what each of these lines do and why its like this for beginners